Samples/excel-open-in-teams/README.md

```diff
@@ -0,0 +1,250 @@
+    - In the **Redirect URI** section, ensure that **Web** is selected in the dropdown and then set the URI to `https://localhost:44326/`.
+    
+    **Note:** The port number used for the redirect URI (`44326`) must match the port your web server is running on. When you open the Visual Studio solution in later steps, you can find the web server's port number by selecting the **ContosoWebApp** project in **Solution Explorer**, then looking at the **SSL URL** setting in the properties window.
+
+1. Choose **Register**.
+
+1. On the **OpenInTeamsSample** page, copy and save the **Application (client) ID**. You'll use it in later procedures.
+
+1. Under **Manage**, select **Authentication**. Under **Implicit grant**, check the following checkboxes.
+    - **Access tokens**
+    - **ID tokens**
+1. Choose **Save** to save the previous authentication changes.
+
+1. Under **Manage**, select **Certificates & secrets**. Select the **New client secret** button. Enter a value for **Description**, then select an appropriate option for **Expires**, and choose **Add**.
+
+1. Copy and save the client secret value. You'll use it in later procedures.
+
+1. Under **Manage**, select **API permissions** and then select **Add a permission**. On the panel that opens, choose **Microsoft Graph** and then choose **Delegated permissions**.
+
+1. Use the **Select permissions** search box to search the following permissions:
+
+    - Channel.ReadBasic.All
+    - ChannelMessage.Send
+    - Files.ReadWrite.All
+    - openid
+    - profile
+    - Team.ReadBasic.All
+
+    **Note:** The `User.Read` permission may already be listed by default. It's a good practice not to ask for permissions that are not needed, so we recommend that you uncheck the box for this permission if your add-in does not actually need it.
+
+1. Select the check box for each permission as it appears. After selecting the permissions, select the **Add permissions** button at the bottom of the panel.
+
+1. On the same page, choose the **Grant admin consent for [tenant name]** button, and then select **Yes** for the confirmation that appears.
+
+    **Note:** After choosing **Grant admin consent for [tenant name]**, you may see a banner message asking you to try again in a few minutes so that the consent prompt can be constructed. If so, you can start work on the next section, ***but don't forget to come back to the portal and press this button***!
+
+## Configure the Sample
+
+Before you run the sample, you'll need to do a few things to make it work properly.
+
+1. In Visual Studio, open the **excel-open-in-teams.sln** solution file for this sample.
+1. In the **Solution Explorer**, open **ContosoWebWeb > Web.config**.
+1. Replace the `[Application (client) ID]` value in both places where it appears with the application ID you generated as part of the app registration process.
```

Comment:
It's actually only in the web.config in one place.
```suggestion
1. Replace the `[Application (client) ID]` value with the application ID you generated as part of the app registration process.
```

Reply:
This is fixed now.